src/gpo/assets/PowerToys.admx

```diff
@@ -319,7 +319,27 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
-	<policy name="DisableAutomaticUpdateDownload" class="Both" displayName="$(string.DisableAutomaticUpdateDownload)" explainText="$(string.DisableAutomaticUpdateDownloadDescription)" key="Software\Policies\PowerToys" valueName="AutomaticUpdateDownloadDisabled">
+	  <policy name="DisablePerMachineInstallation" class="Both" displayName="$(string.DisablePerMachineInstallation)" explainText="$(string.DisablePerMachineInstallationDescription)" key="Software\Policies\PowerToys" valueName="PerMachineInstallationDisabled">
+      <parentCategory ref="InstallerUpdates" />
+      <supportedOn ref="SUPPORTED_POWERTOYS_0_69_0" />
+      <enabledValue>
+        <decimal value="1" />
+      </enabledValue>
+      <disabledValue>
+        <decimal value="0" />
+      </disabledValue>
+    </policy>
+	  <policy name="DisablePerUserInstallation" class="Both" displayName="$(string.DisablePerUserInstallation)" explainText="$(string.DisablePerUserInstallationDescription)" key="Software\Policies\PowerToys" valueName="PerUserInstallationDisabled">
```

Comment:
Set class to "Machine"? Imo doing this only for some users on a computer makes no sense.

If yes, we should update readme, explanation and maybe code.

Same for other policy.